book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author, :rentals
+
+  def add_rental(date, person)
+    @rentals << Rental.new(self, person, date)
```

Comment:
- [ ] **[OPTIONAL]** Pushing an instance of rental to this `@rentals` variable is unnecessary, The initialize method in the **`Rental`** class already takes care of that.

  I would recommend you have something more concise like:
  ```
  Rental.new(self, person, date)
  ```